README.md

```diff
@@ -127,23 +127,44 @@ DSPACE_REST_NAMESPACE # The namespace of the REST application
 DSPACE_REST_SSL # Whether the angular REST uses SSL [true/false]
 ```
 
+All other settings can be set using the following convention for naming the environment variables:
+
+1. replace all `.` with `_`
+2. convert all characters to upper case
+
+e.g.
+
+```
+cache.msToLive.default => CACHE_MSTOLIVE_DEFAULT
+auth.ui.timeUntilIdle => AUTH_UI_TIMEUNTILIDLE
+```
+
 The same settings can also be overwritten by setting system environment variables instead, E.g.:
 ```bash
 export DSPACE_HOST=api7.dspace.org
 ```
 
-The priority works as follows: **environment variable** overrides **variable in `.env` file** overrides **`environment.(prod, dev or test).ts`** overrides **`environment.common.ts`**
+The priority works as follows: **environment variable** overrides **variable in `.env` file** overrides external config set by `APP_CONFIG_PATH` overrides **`config.(prod or dev).yml`**
+
+These configuration sources are collected **at run time**, and written to `dist/browser/assets/config.json` for production and `src/app/assets/config.json` for development.
 
-These configuration sources are collected **at build time**, and written to `src/environments/environment.ts`.  At runtime the configuration is fixed, and neither `.env` nor the process' environment will be consulted.
+The configuration file can be externalized by using environment variable `APP_CONFIG_PATH`.
```

Comment:
Again, would rather this be prefixed with DSPACE.  So `DSPACE_APP_CONFIG_PATH`... or maybe rename to `DSPACE_UI_CONFIG_PATH` as this config is specific to the DSpace user interface (the word "APP" here doesn't make much sense, as DSpace has a backend app as well)

Reply:
I think we need to be able to distinguish environment variables between these two uses.

Reply:
@wwelling : Again, in this README, you forgot to update `APP_CONFIG_PATH` -> `DSPACE_APP_CONFIG_PATH`.  See lines 164 and 168

Reply:
Updated.